spec/tapioca/cli_spec.rb

```diff
@@ -57,29 +57,32 @@ def run(command, args = [], flags = {})
     }.merge(flags).flat_map { |k, v| ["--#{k}", v.to_s] }
 
     exec_command = [
+      "bundle",
+      "exec",
       "bin/tapioca",
       command,
       *flags,
       *args,
     ]
 
-    IO.popen(
-      {
-        "BUNDLE_GEMFILE" => (repo_path / "Gemfile").to_s,
-      },
-      exec_command.join(' '),
-      chdir: repo_path
-    ).read
+    Bundler.with_clean_env do
+      IO.popen(
+        exec_command.join(' '),
+        chdir: repo_path
+      ).read
+    end
   end
 
   before(:all) do
     @repo_path = (Pathname.new(__dir__) / ".." / "support" / "repo").expand_path
-    IO.popen(
-      {
-        "BUNDLE_GEMFILE" => (@repo_path / "Gemfile").to_s,
-      },
-      ["bundle", "install", "--quiet"]
-    ).read
+    Bundler.with_clean_env do
+      IO.popen(
+        {
+          "BUNDLE_GEMFILE" => (@repo_path / "Gemfile").to_s,
+        },
+        ["bundle", "install", "--quiet"]
+      ).read
```

Comment:
Can we not run this in `@repo_path` too then? That way we can drop the `BUNDLE_GEMFILE` from this call too.

Reply:
Good idea, done